srv/modules/runners/validate.py

```diff
@@ -593,21 +616,25 @@ def pillars(**kwargs):
     #options = SaltOptions()
     local = salt.client.LocalClient()
     cluster = ClusterAssignment(local)
+    
+    printer = printer = get_printer(**kwargs)
+
 
     for name in cluster.names:
-        pillar(name, **kwargs)
+        pillar(name, printer=printer, **kwargs)
+    
+    printer.print_result()
 
 
-def pillar(name = None, **kwargs):
+def pillar(name = None, printer=None, **kwargs):
     """
     Check that the pillar for each cluster meets the requirements to install
     a Ceph cluster.
     """
-
-    if name == None:
-        if 'name' in kwargs:
-            name = kwargs['name']
-
```

Comment:
I really didn't thought about the way `validate.pillar` is called by anyone. 

 I was just thinking about the Python function on it's own. If you're declaring a Python function (e.g. `print_args`) with some regular arguments (e.g. `name`) plus a star-star-argument for the rest of the function arguments (e.g. `kwargs`), a regular argument value will never end up in the star-star-argument. 

This is completely unrelated to the way how salt is calling your function, I just saw an `if`-expression that was always evaluated to `False`. Thus, I removed it.